mailet/standard/src/test/java/org/apache/james/transport/mailets/OnlyTextTest.java

```diff
@@ -88,7 +90,9 @@ public void testService() throws MessagingException, IOException {
         message.setContent(mp);
         message.saveChanges();
 
-        mail = new FakeMail(message);
+        mail = FakeMail.builder()
+                .mimeMessage(message)
+                .build();
```

Comment:
In these (test) cases, I'm not sure the builder is more clear neither more simple to write